packages/frontend/src/lib/disk-image/DiskImageDetailsSummary.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="ts">
+import Table from '/@/lib/upstream/DetailsTable.svelte';
+import Title from '/@/lib/upstream/DetailsTitle.svelte';
+import type { BootcBuildInfo } from '/@shared/src/models/bootc';
+import Cell from '/@/lib/upstream/DetailsCell.svelte';
+import Link from '../Link.svelte';
+
+export let image: BootcBuildInfo | undefined;
+</script>
+
+<Table>
+  <tr>
+    <Title>Details</Title>
+  </tr>
+  {#if image}
+    <tr>
+      <Cell>Source image</Cell>
+      <Cell>{image.image}:{image.tag}</Cell>
```

Comment:
Something nice (follow up PR) could be done here, is to have the image as a link and [redirect to the image](https://github.com/containers/podman-desktop/blob/208b2a07fc172213fc5b3ee143e463a24fbfe597/packages/extension-api/src/extension-api.d.ts#L4684-L4685) details page.

Reply:
Good idea! Yes, I'll do this via another PR.